README.md

```diff
@@ -591,6 +591,16 @@ configured to operate in IPv6 mode. Prefix delegation is only supported on nitro
 
 ---
 
+#### `ENABLE_NFTABLES` (v1.13.0+)
+
+Type: Boolean as a String
+
+Default: `false`
+
+VPC CNI uses `iptables-legacy` by default. Setting `ENABLE_NFTABLES` to `true` will update VPC CNI to use `iptables-nft`.
+
+**Note:** VPC CNI image contains `iptables-legacy` and `iptables-nft`. Switching between them is done via `update-alternatives`. It is recommended that the iptables mode matches that which is used by the base OS and kube-proxy.
```

Comment:
Any callouts - Switch between modes while there are rules or pods running?

Reply:
Yeah, good point. No cleanup/switchover happens when you switch modes, so customer would either need to manually update rules or reload node

Reply:
Updated README with some strong language. Unfortunately, switching is not easy on a running system. This works best with a greenfield deployment.